components/brave_wallet_ui/constants/locale.ts

```diff
@@ -295,6 +295,11 @@ const locale = {
   connectWithSiteNext: 'Next',
   connectWithSiteHeaderTitle: 'Connect With Brave Wallet',
 
+  // Import from Hardware
```

Comment:
it is not from hardware though, let's just say external wallets

Reply:
Updated!